src/components/Dashboard/Wishlist/__tests__/ProductInWhislist.test.js

```diff
@@ -19,8 +20,13 @@ describe('ProductInWishlist />', () => {
     expect(wrapper).toMatchSnapshot();
   });
   it('should call remove after click', () => {
-    const { wrapper, props } = setup();
+    const {
+      wrapper,
+      props: {
+        product: { productId },
+      },
+    } = setup();
     wrapper.find('button').simulate('click');
-    expect(wrapper.instance().props.remove).toHaveBeenCalledWith(props.product.productId);
+    expect(wrapper.instance().props.remove).toHaveBeenCalledWith(productId);
```

Comment:
What about accessing props destructured from `setup`? Instead of `wrapper.instance().props`

Here and in other places